### PR TITLE
revisions to the existing publications entries

### PR DESCRIPTION
**Pull request by Philip-Schaer**
Changed publication formatting from authors-then-title to title-then-authors for all currently added publications, added a hyperlink to each paper (journal version or arXiv), added a link to published source code where applicable

### CODE REVIEW

File: _publications/2023-01-19-gsss.md [hunk 1]
```diff
@@ -4,7 +4,8 @@ involved:
...
 ---
 
-Michael Habeck, Mareike Hasenpflug, Shantanu Kodgirwar, Daniel Rudolf  
 *Geodesic slice sampling on the sphere*  
-arXiv preprint arXiv:2301.08056, 2023
-
+Michael Habeck, Mareike Hasenpflug, Shantanu Kodgirwar, Daniel Rudolf  
+arXiv preprint arXiv:2301.08056, 2023  
+[Link to Paper](https://arxiv.org/abs/2301.08056)  
+[Link to Source Code](https://github.com/michaelhabeck/spherical-slice-sampling)
```
Comment (andreasKroepelin) [inline _publications/2023-01-19-gsss.md:11]: This link doesn't work for me.
(Is the repository private?)

**Review by andreasKroepelin — COMMENTED** [review]
One code link did not work for me, all other links seem to do what they're supposed to do.

File: _publications/2023-01-19-gsss.md (hunk 1 above)
Comment (Philip-Schaer) [reply _publications/2023-01-19-gsss.md:11]: Ah you're right, I got the wrong one.

Comment (Philip-Schaer) [reply _publications/2023-01-19-gsss.md:11]: It should be https://github.com/ShantanuKodgirwar/geosss but I can't figure out how to change that without creating a separate commit :see_no_evil:

Comment (trahflow) [reply _publications/2023-01-19-gsss.md:11]: it's fine to create a separate commit.

Comment (Philip-Schaer) [reply _publications/2023-01-19-gsss.md]: Ah, it just shows up here as an additional commit!

Comment (Philip-Schaer) [reply _publications/2023-01-19-gsss.md]: Seeing as Andreas already tested everything else, I think I'll just merge this myself now :grin:

**Pull request merged**